s30566-DanielMakowski/zajecia1/instalacja_debian_linux.md

```diff
@@ -437,26 +462,70 @@ x.  Interfejs Użytkownika -- Ta sekcja umożliwia zmianę pewnych aspektów
         razu zbootować instalację debiana. Dostępne mamy 5 opcji:
 
         i.  Instalacja graficzna
+        		1. Graficzna, prosta metoda instalacji wspierająca instalację
+        		przy użyciu klawiatury i myszy.
 
         ii. Instalacja (w prostszej formie)
+        		1. Instalacja z prostym motywem graficznym, zdecydowanie mniej
+        		wymagająca, wspierająca jedynie wejście klawiatury.
 
         iii. Ustawienia Zaawansowane
 
              1.  Graficzna instalacja w trybie eksperta
+             	a) Tryb instalacji wyszczególniający wszystkie możliwe
+             	opcje w procesie instalacji. Zezwala również na nawigację
+             	między różnymi krokami, które nie wykonują się automatycznie
+             	w domyślnej sekwencji. Wspiera mysz oraz klawiaturę.
+             		
 
              2.  Graficzny tryb odzyskiwania
+             	a) Graficzny tryb odzyskiwania służy do przywracania 
+             	uszkodzonej instalacji systemu lub do naprawienia bootloadera. 
+             	Pokazane zostanie kilka pierwszych ekranów instalatora w 
+             	bardziej zaawansowanej grafice niż ta, jaką uświadczymy w 
+             	przypadku zwykłego trybu odzyskiwania z informacją, że jest to 
+             	tryb odzyskiwania. Obsługuje wejście z klawiatury i myszy. Tryb 
+             	ten przejmuje "dowodzenie" nad hardware'em aby zapewnić, że 
```

Comment:
ciekawy sposób ujęcia tego zagadnienia.
Ale tak naprawdę będzie to wykrycie sprzętu i zamontowanie dostępnych systemów plików